### ISSUE TITLE
Improve keyboard accessibility of pulldown nav menu

### ISSUE THREAD

**Issue by cjg89** [UCF/Pegasus-Theme#123]
We should probably look at replacing the existing pulldown menu logic to use an Athena navbar + collapse buttons, and incorporate relevant aria attributes to improve the user experience for keyboard and screenreader users.

Additionally, we need to ensure each link in the nav bar has a visible focus state--currently, the Pegasus logo's outline is not visible.
